docs/openj9_defaults.md

```diff
@@ -82,7 +82,7 @@ The last 2 columns show whether the default setting can be changed by a command-
 |Maximum stack size for Java Threads **(32-bit)**. Use `-Xss<size>`  |320 KB    |320 KB    |N/A              |320 KB           |320 KB    |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
 |Maximum stack size for Java Threads **(64-bit)**. Use `-Xss<size>`  |1024 KB   |1024 KB   |1024 KB          |1024 KB          |1024 KB   |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
 |Stack size for OS Threads **(32-bit)**. Use `-Xmso<size>`           |256 KB    |256 KB    |N/A              |32 KB            |256 KB    |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
-|Stack size for OS Threads **(64-bit)**. Use `-Xmso<size>`     |512 KB    |![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB (256 KB on IBM Z&reg;) ![End of content that applies only to Java 17 (LTS) and later](cr/java_close_lts.png)<br/><br/>Earlier Java versions: 256 KB (512 KB on PPC) |![Start of content that applies to Java 17 plus](cr/java17plus.png) Architecture: x86: 512 KB, AArch64 (Apple silicon): 256 KB ![End of content that applies only to Java 17 (LTS) and later](cr/java_close_lts.png)<br/><br/>Earlier Java versions: 256 KB |![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB ![End of content that applies only to Java 17 (LTS) and later](cr/java_close_lts.png)<br/><br>Earlier Java versions: 256 KB  |1 MB      |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
+|Stack size for OS Threads **(64-bit)**. Use `-Xmso<size>`     |512 KB    |![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB (256 KB on IBM Z&reg;) ![End of content that applies only to Java 17 (LTS) and later](cr/java_close_lts.png)<br/><br/>AArch64: 256 KB<br/><br/>Earlier Java versions: 256 KB (512 KB on PPC) |![Start of content that applies to Java 17 plus](cr/java17plus.png) Architecture: x86: 512 KB, AArch64 (Apple silicon): 256 KB ![End of content that applies only to Java 17 (LTS) and later](cr/java_close_lts.png)<br/><br/>Earlier Java versions: 256 KB |![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB ![End of content that applies only to Java 17 (LTS) and later](cr/java_close_lts.png)<br/><br>Earlier Java versions: 256 KB  |1 MB      |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
```

Comment:
Seems the update should be in the brackets with "256 KB on IBM Z".

Reply:
I see, it works for alinux either way, it could be in the brackets with zlinux or not. 

The information for -Xmso linux seems overly complicated, it can be clarified. The jdk17+ change is specific to xlinux.

xlinux, 8, 11: 256K 17+: 512k
plinux always 512k
zlinux always 256k
alinux always 256k